selfdrive/athena/athenad.py

```diff
@@ -28,25 +29,32 @@
 from selfdrive.loggerd.config import ROOT
 from selfdrive.swaglog import cloudlog
 
-ATHENA_HOST = os.getenv('ATHENA_HOST', 'wss://athena.comma.ai')
+ATHENA_HOST = os.getenv('ATHENA_HOST', 'wss://athena.comma.ai:8765')
```

Comment:
should not have changed

Reply:
Ops, it's a leftover of a fix that I introduce to allow local development. I don't have push access anymore so I cannot push the fix on this branch